src/lib/style/abstracts/_variables.scss

```diff
@@ -47,10 +47,10 @@ $bootstrap-4-danger-bg: #f2dede !default;
 $bootstrap-4-danger-text-color: rgb(102, 102, 102) !default;
 $space-unit: 2.1rem;
 
-$green-affirmative: #35bf7d !default;
-$yellow-warning: #f0dc26 !default;
+$green-affirmative: map.get($colors, 'success') !default;
```

Comment:
Adding some consistency, using the colors from the new palette instead of unique colors.